evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -143,32 +143,19 @@ def test_make_pipeline(
                 else []
             )
             natural_language_featurizer = (
-                [NaturalLanguageFeaturizer]
-                if "text" in column_names and input_type == "ww"
-                else []
+                [NaturalLanguageFeaturizer] if "text" in column_names else []
```

Comment:
why is this needed?

Reply:
New natural language inference means even with pandas input, we'll infer natural language! Before, it was inferred as Unknown, hence the DropColumn transformer that got removed.

Reply:
sounds great, thanks!